### PR TITLE
Fix text displacement when there are no reviews

### PR DESCRIPTION
**Pull request by Wopian**
AND:

- Shorter message for when there are no reviews,

- Remove `meh` icon on anime index's review,

- Hide review links when there are no reviews,
  - Write One => Duplicate link of button added when there are no reviews,
  - View All => Only takes you to an empty page when there are no reviews.

### CODE REVIEW

File: app/assets/stylesheets/partials/anime-page.css.scss [hunk 1]
```diff
@@ -519,6 +519,8 @@
       margin-bottom: 35px;
       padding: 25px;
       padding-bottom: 0;
+      display: inline-flex;
```
Comment (NuckChorris) [inline app/assets/stylesheets/partials/anime-page.css.scss]: I remember having trouble with flexbox and our prefixer not playing along back with the chat.  I'd double check that this works in all browsers and has the necessary prefixes.

Also flexbox stronk

**Comment by NuckChorris** [comment]
Should probably rebase these down, it's 6 commits for 11 lines of changes lol

File: app/assets/stylesheets/partials/anime-page.css.scss (hunk 1 above)
Comment (Wopian) [inline app/assets/stylesheets/partials/anime-page.css.scss]: :+1:

**Comment by vikhyat** [comment]
lgtm, can merge once rebased

**Comment by Wopian** [comment]
brb killing git.

**Pull request closed without merging**